gen/go.mod

```diff
@@ -6,7 +6,7 @@ replace (
 	// As gen schema generates a schema for a chart via reflect, we
 	// want it to always operate on the current chart version.
 	github.com/redpanda-data/redpanda-operator/charts/operator => ../charts/operator
-	github.com/redpanda-data/redpanda-operator/charts/redpanda => ../charts/redpanda
+	github.com/redpanda-data/redpanda-operator/charts/redpanda/v5 => ../charts/redpanda
```

Comment:
Linter complains about this import https://buildkite.com/redpanda/redpanda-operator/builds/4743#019580c7-607c-4f55-b2cf-881be848aa0d/225-744 probably regexp needs to be adjusted.

Reply:
Fixed!